twake/backend/preview/src/previews/api.ts

```diff
@@ -0,0 +1,30 @@
+export interface PreviewServiceAPI {
+  /**
+   * Save a file and returns its entity
+   *
+   * @param id
+   * @param file
+   * @param options
+   * @param context
+   */
+  save(id: string): Promise<File>;
+
+  /**
+   * Get a file as readable and metadata information from its ID
+   *
+   * @param id
+   * @param context
+   */
+  download(
+    id: string,
+    context: any
```

Comment:
You can create a temporary type and use him here  `type ContextType =  { [key:string]: any };`